src/crab/cfg.hpp

```diff
@@ -12,6 +12,7 @@
  * variable.
  *
  */
+#include <map>
 #include <memory>
 #include <map>
```

Comment:
Why include `<map>` twice?

Reply:
Not intentional, looks like a merge/rebase conflict.  I had meant to alphabetize them, i.e., move map not duplicate it.